src/SFA.DAS.PR.Application.UnitTests/EmployerRelationships/Queries/GetEmployerRelationships/GetEmployerRelationshipsQueryValidatorTests.cs

```diff
@@ -0,0 +1,23 @@
+﻿using FluentValidation.TestHelper;
+using SFA.DAS.PR.Application.Permissions.Queries.GetEmployerRelationships;
+
+namespace SFA.DAS.PR.Application.UnitTests.Permissions.Queries.GetAllPermissionsForAccount;
+public class GetEmployerRelationshipsQueryValidatorTests
+{
+    [Test]
+    public async Task Validate_AccountHashedId_Valid_Query()
+    {
+        var sut = new GetEmployerRelationshipsQueryValidator();
+        var result = await sut.TestValidateAsync(new GetEmployerRelationshipsQuery("hash"));
+        result.ShouldNotHaveValidationErrorFor(query => query.AccountHashedId);
+    }
+
+    [Test]
+    public async Task Validate_AccountHashedId_Returns_ErrorMessage()
```

Comment:
Unit test method naming convention <action>_<condition>_<result> (all optional as long as it reads fine) ValidateAccountHashedId_Empty_Invalid. I would also check for null here, you can use TestCase to avoid duplication

Reply:
I've updated the method name, the request itself won't accept a nullable value so no need for a test for this scenario.